src/ucs/stats/stats.c

```diff
@@ -935,12 +919,14 @@ void ucs_stats_init()
 
     /* Aggregate-sum class id to name database initialize */
     ucs_stats_context.aggrgt_cnt_names_db = NULL;
-
-    /* We start from 1, index == 0 is reserved */
-    ucs_stats_context.aggrgt_sum_db_size        = 1;
     ucs_stats_context.aggrgt_sum_db_last_offset = 0;
     ucs_stats_context.aggrgt_cnt_names_db_size  = 0;
 
+    /* Since index starts from 1, add a place-holder element to the array */
+    ucs_assert(
+        ucs_array_append(aggrgt_sum_db_array_t, &ucs_stats_context.aggrgt_sum_db) ==
+        UCS_OK);
```

Comment:
what happens in release mode?

Reply:
Updated to print an error in release mode as well.